apps/client/src/pages/QuestionsPage.js

```diff
@@ -46,34 +50,78 @@ const QuestionsPage = ({ topic, checker }) => {
     });
 
   const onQuestionNext = (value) => {
-    if (question.options) {
+    // Provide the user answers to the `sttr-checker`
+    if (question.options && value) {
       question.setAnswer(value);
-    } else {
+    }
+    if (!question.options && value) {
```

Comment:
How can this even work? Value can be `false` right?

Reply:
On a 'No' answer the value in console.log is `no` and not `false`